### PR TITLE
Bootstrap 3 Migration: Users > Organization Structure

### PR DESCRIPTION
**Pull request by pnaw94**
@biyeun 
scr01
scr02
scr03
scr04
scr05
scr06
scr07

### CODE REVIEW

File: corehq/apps/locations/views.py [hunk 1]
```diff
@@ -261,6 +271,12 @@ class NewLocationView(BaseLocationView):
     creates_new_location = True
     form_tab = 'basic'
 
+    @use_bootstrap3
+    @use_knockout_js
```
Comment (biyeun) [inline corehq/apps/locations/views.py]: please pull / rebase latest master into this branch and remove `use_knockout_js` :)

**Comment by pnaw94** [comment]
@biyeun updated

**Comment by biyeun** [comment]
@gcapalbo can you make a run through of this just to make sure it works as intended? thanks!

**Comment by biyeun** [comment]
Sorry, looks like @esoergel should be the one to take a pass at this. Ethan, can you take a peek? Thank you!

**Comment by esoergel** [comment]
@biyeun will do

File: corehq/apps/locations/templates/locations/manage/locations.html [hunk 2]
```diff
@@ -1,163 +1,104 @@
...
+        var can_edit_root = {{ can_edit_root|yesno:"true,false" }};
+        var selected_loc = {{ selected_id|JSON }};
+        var hierarchy = {{ hierarchy|JSON }};
+        var show_inactive = {{ show_inactive|yesno:"true,false" }};
+
+        function loc_edit_url(loc_id) {
+            var template = '{% url "edit_location" domain "-locid-" %}';
+            return template.replace('-locid-', loc_id);
+        }
     }
```
Comment (esoergel) [inline corehq/apps/locations/templates/locations/manage/locations.html]: I tried running this locally and got a failure on this extra `}`, I think it just needs to be removed.

File: corehq/apps/locations/templates/locations/manage/import.html [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 
 {% block js %} {{ block.super }}
-    <script src="{% static 'commtrack/ko/location_bulk_upload_file.js' %}"></script>
+    <script src="{% new_static 'commtrack/ko/location_bulk_upload_file.js' %}"></script>
 {% endblock %}
 
-{% block main_column %}
-    {% include "locations/manage/partials/bulk_upload.html" %}
+{% block page_content %}
+    {% include "style/bootstrap3/partials/bulk_upload.html" %}
```
Comment (esoergel) [inline corehq/apps/locations/templates/locations/manage/import.html]: The original template `"locations/manage/partials/bulk_upload.html"` includes a couple extra options for the download which are removed here.  That page extendes `"style/bootstrap2/partials/bulk_upload.html"` - I think you should be able to just switch it over to bootstrap 3 there, but we do need those checkboxes.

File: corehq/apps/locations/templates/locations/manage/locations.html [hunk 4]
```diff
@@ -1,163 +1,104 @@
...
-
-function loc_unarchive_url(loc_id) {
-    var template = '{% url "unarchive_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
-
-function loc_delete_url(loc_id) {
-    var template = '{% url "delete_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
```
Comment (esoergel) [inline corehq/apps/locations/templates/locations/manage/locations.html]: There is a lot of code that's being incorrectly deleted here.  If you view the diff with whitespace ignored, it's more clear: https://github.com/dimagi/commcare-hq/pull/9862/files?w=1

It looks like all the code about deleting locations will be removed by this PR, my guess is there was a bad merge.

File: corehq/apps/locations/views.py [hunk 5]
```diff
@@ -76,6 +77,12 @@ class LocationsListView(BaseLocationView):
     page_title = ugettext_noop("Organization Structure")
     template_name = 'locations/manage/locations.html'
 
+    @method_decorator(can_edit_location_types)
```
Comment (esoergel) [inline corehq/apps/locations/views.py]: Why is this being added?

**Comment by pnaw94** [comment]
@esoergel sorry, I must have rushed the merge
All of those should be resolved now

File: corehq/apps/locations/forms.py [hunk 6]
```diff
@@ -96,6 +96,9 @@ def __init__(self, location, bound_data=None, is_new=False, user=None,
 
         self.custom_data = self.get_custom_data(bound_data, is_new)
 
+        self.custom_data.form.helper.label_class = 'col-sm-3 col-md-4 col-lg-2'
+        self.custom_data.form.helper.field_class = 'col-sm-4 col-md-5 col-lg-3'
```
Comment (esoergel) [inline corehq/apps/locations/forms.py]: I think there's something wrong with this part - when I view a location edit form locally with custom data - it doesn't show up as an editable field.  To reproduce, go to `/a/<domain>/settings/locations/fields/` and add a field (it doesn't matter what).  Then go to a location page `/a/<domain>settings/locations/<location_id>/` and observe that the "Additional Information" section at the bottom is broken.  Here's what it looks like in production: https://www.commcarehq.org/a/ethan-commtrack/settings/locations/c2b96890a06a3d4d5e0a024c42ceeef6/

Comment (esoergel) [inline corehq/apps/locations/forms.py]: Actually, I'm mistaken - it's the "Default monthly consumption values" section that is broken.

**Comment by pnaw94** [comment]
@esoergel updated

**Pull request merged**